editor/src/clj/editor/boot_open_project.clj

```diff
@@ -175,9 +175,8 @@
           scene-visibility     (scene-visibility/make-scene-visibility-node! *view-graph*)
           app-view             (app-view/make-app-view *view-graph* project stage menu-bar editor-tabs-split tool-tabs prefs)
           outline-view         (outline-view/make-outline-view *view-graph* project outline app-view)
-          properties-view      (properties-view/make-properties-view workspace project app-view *view-graph* (.lookup root "#properties"))
           asset-browser        (asset-browser/make-asset-browser *view-graph* workspace assets prefs)
-          open-resource        (partial app-view/open-resource app-view prefs workspace project)
+          open-resource        (partial #'app-view/open-resource app-view prefs workspace project)
```

Comment:
I used a var instead of a function reference here so it can be redefined at the REPL.